control/static/index.html

```diff
@@ -88,24 +88,24 @@ <h2>Column 3</h2>
-      <li class="nav-item" role="presentation"><a class="nav-link" aria-selected="False"          accesskey="s" data-bs-toggle="tab" type="button" role="tab" aria-controls="visualisation" data-bs-target="#visualisation">&nbsp;&nbsp;Visualisation</a></li>
+      <li class="nav-item" role="presentation"><a class="nav-link active" aria-selected="False"          accesskey="s" data-bs-toggle="tab" type="button" role="tab" aria-controls="visualisation" data-bs-target="#visualisation">&nbsp;&nbsp;Visualisation</a></li>
 
       <!-- <li class="nav-item" role="presentation"><a class="nav-link active" aria-selected="True"    accesskey="t" data-bs-toggle="tab" type="button" role="tab" aria-controls="processing" data-bs-target="#processing">Processing</a></li> -->
       <li class="nav-item" role="presentation"><a class="nav-link" aria-selected="False"    accesskey="t" data-bs-toggle="tab" type="button" role="tab" aria-controls="processing" data-bs-target="#processing">Processing</a></li>
 
       <!-- <li class="nav-item" role="presentation"><a class="nav-link active" aria-selected="True"          accesskey="u" data-bs-toggle="tab" type="button" role="tab" aria-controls="health" data-bs-target="#health">Health</a></li> -->
       <li class="nav-item" role="presentation"><a class="nav-link" aria-selected="False"          accesskey="u" data-bs-toggle="tab" type="button" role="tab" aria-controls="health" data-bs-target="#health">Health</a></li>
 
-      <li class="nav-item" role="presentation"><a class="nav-link active" aria-selected="False"          accesskey="o" data-bs-toggle="tab" type="button" role="tab" aria-controls="debug" data-bs-target="#debug">Debug</a></li>
+      <li class="nav-item" role="presentation"><a class="nav-link " aria-selected="False"          accesskey="o" data-bs-toggle="tab" type="button" role="tab" aria-controls="debug" data-bs-target="#debug">Debug</a></li>
       <!-- <li class="nav-item" role="presentation"><a class="nav-link" aria-selected="False"          accesskey="o" data-bs-toggle="tab" type="button" role="tab" aria-controls="debug" data-bs-target="#debug">Debug</a></li> -->
 
       <!-- <li class="nav-item" role="presentation"><a class="nav-link active" aria-selected="True"                  data-bs-toggle="tab" type="button" role="tab" aria-controls="logging" data-bs-target="#logging">Logging</a></li> -->
       <li class="nav-item" role="presentation"><a class="nav-link" aria-selected="False"                        data-bs-toggle="tab" type="button" role="tab" aria-controls="logging" data-bs-target="#logging">Logging</a></li>
     </ul>
 
     <div class="tab-content"> <!-- tab-content -->
-      <div class="tab-pane fade" id="visualisation" role="tabpanel">
-      <!-- <div class="tab-pane fade show active" id="visualisation" role="tabpanel"> -->
+      <!--<div class="tab-pane fade" id="visualisation" role="tabpanel">-->
```

Comment:
Maintain space before final tag: `role="tabpanel">-->` should be `role="tabpanel"> -->`